packages/emotion-theming/types/tslint.json

```diff
@@ -1,21 +1,22 @@
 {
-    "extends": "dtslint/dtslint.json",
-    "rules": {
-        "array-type": [true, "generic"],
-        "semicolon": false,
-        "whitespace": [
-            true,
-            "check-branch",
-            "check-decl",
-            "check-operator",
-            "check-module",
-            "check-rest-spread",
-            "check-type",
-            "check-typecast",
-            "check-type-operator",
-            "check-preblock"
-        ],
-        "no-null-undefined-union": false,
-        "no-unnecessary-generics": false
-    }
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "array-type": [true, "generic"],
+    "semicolon": false,
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-operator",
+      "check-module",
+      "check-rest-spread",
+      "check-type",
+      "check-typecast",
+      "check-type-operator",
+      "check-preblock"
+    ],
+    "no-null-undefined-union": false,
+    "no-unnecessary-generics": false,
+    "callable-types": false
```

Comment:
Could we unify all of existing tslint.json configs?

Reply:
I'd prefer that to be a separate PR to be honest. Unfortunately the tests being broken has caused this PR to get a little out of scope.

Reply:
Sure, no problem.